src/pages/Products/ProductListing.jsx

```diff
@@ -1,7 +1,128 @@
 import React from "react";
+import "./ProductListing.css";
+import { ProductCard } from "./components/ProductCard";
+import { chelseaTee } from "../../assets";
+
+import { useProduct } from "../../context/data/ProductContext";
 
 const ProductListing = () => {
-  return <div>Products</div>;
+  const { products } = useProduct();
+
+  return (
+    <div>
+      <main className="product-main-container">
+        <aside className="product-filter-container">
+          <div className="filter-header">
+            <h3 className="filter-title">Filter</h3>
+            <p className="filter-reset">Clear</p>
+          </div>
+          <div className="price-range-container">
+            <h3 className="price-title">Price</h3>
+            <div className="slider-container">
+              <div className="price-range">
+                <p>0</p>
+                <p>2500</p>
+                <p>5000</p>
+              </div>
+              <input
+                type="range"
+                name="rangeInput"
+                className="slider"
+                min="0"
+                max="5000"
+              />
+            </div>
+          </div>
+          <div className="category-container">
+            <h3 className="category-title">Category</h3>
+            <div className="checkbox">
+              <label for="checkbox-1">
```

Comment:
use `htmlFor` instead of for atrribute here

Reply:
I am going to break the sidebar into small components in next pr will do that then.Thanks for the suggestion